app/controllers/schedules_controller.rb

```diff
@@ -11,6 +13,8 @@ def show
     @schedule = Schedule.find(params[:id])
     @staffs = Staff.all
     @staff = Staff.new
+    @shifts = Shift.all
+    @date_array = get_schedule_header(@schedule)        
   end
 
   def create
```

Comment:
What does the SchedulesHelper do?

Reply:
It adds all the dates in a given month into an array that is available for the schedule view to use when iterating over shifts and assignments.